CIP-0061/README.md

```diff
@@ -0,0 +1,77 @@
+---
+CIP: 61
+Title: Stake-based Protocol Governance Restrictions
+Authors: Matthias Benkort <matthias.benkort@cardanofoundation.org> 
+Discussions-To: 
+Comments-Summary: A simple, transitional, veto power for ada holders until decentralized governance.  
+Comments-URI:  
+Status: Proposed 
+Type: Standards Track 
+Created: 2022-06-25
+License: CC-BY-4.0
+---
+
+## Abstract
+
+The Cardano journey was divided into several phases, the last of which relates to the decentralization of the network governance. We call this last phase: 'Voltaire'. While Voltaire is in the making, some aspects of the protocol remain relatively centralized and under the control of a few entities. This proposal improves on the current status quo by adding a layer of control to the Stake Pool Operators (a.k.a. SPOs) and by extension, to delegators. This proposal is fairly lightweight and can be implemented without a hard fork. 
+
+## Motivation
+
+Decentralized governance is a hard topic and it is likely to take some time to get to a point where Cardano has a fully working decentralized governance. In the meantime, while tolerated until now, the current status quo is judged unacceptable by many. As of today, the number of entities that can control the current set of governance operations is limited. Those operations include:
+
+- Protocol parameters updates
+- Protocol versions updates
```

Comment:
technically the protocol version is a protocol parameter, but probably nice to spell this out. maybe mention the redundancy, though?

Reply:
Yes, they are indeed part of the same structure, but I thought they were usually quite separate events in nature and thus, worth separating (same goes for MIR actually, which I have separated but are technically the same operation).